nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/json/WriteJsonResult.java

```diff
@@ -376,6 +377,9 @@ private void writeValue(final JsonGenerator generator, final Object value, final
             case STRING:
                 generator.writeString(coercedValue.toString());
                 break;
+            case BIGDECIMAL:
+                generator.writeNumber((BigDecimal) coercedValue);
```

Comment:
We might want
```java
                generator.writeNumber(DataTypeUtils.toBigDecimal(coercedValue, fieldName));
```
just to follow conventions. I think `coercedValue` can only be `BigDecimal` at this point but not sure if that is by design or because of the current implementation.